addon/components/one-way-input.js

```diff
@@ -86,13 +86,12 @@ export default Component.extend({
     return input;
   },
 
+  didInitAttrs() {
```

Comment:
Can you tweak this to use `init` instead? We will likely be deprecating `didInitAttrs` (it provides roughly no value over just using `init`) in the future.